### ISSUE TITLE
training dataset

### ISSUE THREAD

**Issue by Yaffa1607** [MarvinTeichmann/tensorflow-fcn#10]
Hi , what is the format of the data for training?
I have a folder of images to train. How do I pass it through training?

**Comment by MarvinTeichmann** [comment]
Training code is coming soon. Stay tuned. The given code currently justs builds the computational graph. So you can use any data format you like as long as you are able to implement read/write for it and the corresponding input / validation pipeline.

**Comment by MarvinTeichmann** [comment]
Code will be properly published after CVPR deadline.

**Comment by Yaffa1607** [comment]
Hi Marvin ,

For fine tuning the network , should the labels be an array of 1's and 0's
for the corresponding images?

**Comment by Yaffa1607** [comment]
Hi , I am having a looot of trouble trying to train it. Do the images need
labels/ image masks  to train?
Or can I pass a batch of images and it willl automatically detect the part?
I am trying to detect hands in images.
Can you please tell me how to structure the image folder and pass it to
train  in the vgg.build?
